docs/modules/plugins/partials/generic-ui-steps.adoc

```diff
@@ -405,3 +405,51 @@ When I find >= `0` elements `xpath(//*[@class='menu enabled'])` and while they e
 |step                                         |
 |When I click on element located `id(disable)`|
 ----
+
+=== Execute JavaScript with arguments
+
+Executes JavaScript via https://www.selenium.dev/selenium/docs/api/java/org/openqa/selenium/JavascriptExecutor.html[JavascriptExecutor]. Step executes the script either without any arguments or with `String` or `Object` argument types.
+
+[source,gherkin]
+----
+When I execute javascript `$script` with arguments:$arguments
+----
+
+* `$script` - The JavaScript code to execute.
+* `$arguments` - The examples table with set of arguments and their type:
+** [subs=+quotes]`*value*` -  The value of the argument.
+** [subs=+quotes]`*type*` -  The type of the argument. Either `string` or `object`.
+
+.Remove noisy assistant
+[source,gherkin]
+----
+When I execute javascript `document.querySelector('#assistant').remove()` with arguments:
+----
+
+.Inject image
+[source,gherkin]
+----
+When I execute javascript `sauce:inject-image` with arguments:
+|value                                                                                       |type  |
+|iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mNk+A8AAQUBAScY42YAAAAASUVORK5CYII=|string|
```

Comment:
please add some example with object type to make it clear that JSON object is expected